docs/aws/README.md

```diff
@@ -81,15 +89,14 @@ helm upgrade --install karpenter karpenter/karpenter \
   --namespace karpenter --set serviceAccount.create=false
 ```
 
-### (Optional) Enable Verbose Logging
```

Comment:
I think we want to keep this on for the demo.

Reply:
can you send me a quick cmd for someone to view the output? what's an example of something reported only by verbose logging?

Reply:
we talked and, for example, verbose logging includes more aws specific events